src/plugins/irc/irc-config.c

```diff
@@ -3227,6 +3246,30 @@ irc_config_init ()
         NULL, NULL, NULL,
         &irc_config_change_color_item_nick_modes, NULL, NULL,
         NULL, NULL, NULL);
+    irc_config_color_item_tls_version_max = weechat_config_new_option (
+        irc_config_file, ptr_section,
+        "item_tls_version_max", "color",
+        N_("color for max supported tls version in bar item \"tls_version\""),
+        NULL, -1, 0, "green", NULL, 0,
+        NULL, NULL, NULL,
+        &irc_config_change_color_item_tls_version, NULL, NULL,
+        NULL, NULL, NULL);
+    irc_config_color_item_tls_version_1_2 = weechat_config_new_option (
+        irc_config_file, ptr_section,
+        "item_tls_version_1_2", "color",
+        N_("color for TLS1.2 in bar item \"tls_version\""),
+        NULL, -1, 0, "yellow", NULL, 0,
+        NULL, NULL, NULL,
+        &irc_config_change_color_item_tls_version, NULL, NULL,
+        NULL, NULL, NULL);
+    irc_config_color_item_tls_version_other = weechat_config_new_option (
```

Comment:
I would rename this variable `irc_config_color_item_tls_version_disabled` to clearly indicate TLS is disabled (and not "other version").

Reply:
I went with `irc_config_color_item_tls_version_insecure` as it will be used for both cleartext and insecure protocols like TLS1.0